static/app/components/dropdownMenu/index.spec.tsx

```diff
@@ -166,7 +166,6 @@ describe('DropdownMenu', function () {
 
     const parentItem = screen.getByRole('menuitemradio', {name: 'Item'});
 
-    expect(parentItem).toHaveAttribute('aria-haspopup', 'true');
```

Comment:
Removed this check. Looks `useMenuItem()` no longer adds `aria-haspopup`. But it still has `aria-expanded` and the other behavior works fine still.